tests/web/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <!--
+  <head>
```

Comment:
Can you revert formatting for this file, only keep changed lines of code

Reply:
I guess it auto-formatted on push, will revert the auto-formatting.